docs/spanner/usage.rst

```diff
@@ -14,19 +24,77 @@ Spanner
   advanced-session-pool-topics
   api-reference
 
-API requests are sent to the `Cloud Spanner`_ API via RPC over
-HTTP/2.  In order to support this, we'll rely on `gRPC`_.
+Spanner Client
```

Comment:
Do I need to scrutinize this section or is it copy-pasta from the usage doc?

Reply:
The latter. :-)